app/helpers/application_helper.rb

```diff
@@ -14,34 +14,16 @@ def flash_messages
   end
 
   def notification_icon(subject_type)
-    case subject_type
-    when 'RepositoryInvitation'
-      'mail-read'
-    when 'Issue'
-      'issue-opened'
-    when 'PullRequest'
-      'git-pull-request'
-    when 'Commit'
-      'git-commit'
-    end
+    subject_types = {'RepositoryInvitation' => 'mail-read', 'Issue' => 'issue-opened',
+      'PullRequest' => 'git-pull-request', 'Commit' => 'git-commit'}
+
+    subject_types[subject_type]
   end
 
   def reason_label(reason)
-    case reason
-    when 'comment'
-      'primary'
-    when 'author'
-      'success'
-    when 'state_change'
-      'info'
-    when 'mention'
-      'warning'
-    when 'assign'
-      'danger'
-    when 'subscribed'
-      'default'
-    else
-      'default'
-    end
+    reasons = {'comment' => 'primary', 'author' => 'success', 'state_change' => 'info', 'mention' => 'warning',
```

Comment:
Same comment here as above, could be something like.

```ruby
module ApplicationHelper
  REASON_LABELS = {
    'comment'      => 'primary',
    'author'       => 'success',
    'state_change' => 'info',
    'mention'      => 'warning',
    'assign'       => 'danger'
  }.freeze

  def reason_label(reason)
    REASON_LABELS.fetch(reason, 'default')
  end
end
```

And for the `default` you could just use just use [fetch](https://ruby-doc.org/core-2.3.3/Hash.html#method-i-fetch)